app/build.gradle

```diff
@@ -209,6 +210,9 @@ dependencies {
     }
 
     androidTestImplementation "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
+
+    // See https://github.com/mozilla/application-services/blob/master/.buildconfig-android.yml#L1 for latest version
+    testImplementation "org.mozilla.appservices:full-megazord-forUnitTests:0.37.1"
```

Comment:
I used the link and determined there's a newer version as of 23 hours ago. That being said, I don't think it'll break our code to be a little out-of-date so in the name of forward progress, I'm just going to land this.

Reply:
This was my mistake when I spoke with @dnarcese offline to document where the version number should come from.

It should actually follow the version that A-C is using: https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Dependencies.kt#L29

Reply:
@dnarcese Can you fix the link?

Reply:
yup done [here](https://github.com/mozilla-mobile/firefox-tv/pull/2694/commits/b9aad86592ec754ba68e1e604c2a74d3644a8c6b)

Reply:
Also heads up: With the next release we ship a `support-test-appservices` component that will include the right `forUnitTests` version as a transitive dependency so that you do not have to sync them manually anymore.

Reply:
I filed https://github.com/mozilla-mobile/firefox-tv/issues/2738 to do the replacement.